lib/jasmine-2.3.4/boot.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
please don't commit these files, they are so disruptive to have to read through - instead use .gitignore or a CDN.